Implemented Email setup

brazilian-code opened the pull request:
Implemented email for the website using a testing account to send the emails from. Also introduced ADMINS which will receive error logs in their respective emails

brazilian-code committed:
Set up Email

Implemented email settings using a testing account, also introduced admin list, which will receive error messages through email

brazilian-code committed:
updated admins

PineappleBingo reviewed (commented):
Reminder: We should use environment variables to keep your personal info. so that it's not exposed to the public.

PineappleBingo commented on LeftoverIngredients/settings.py:
```diff
@@ -26,9 +26,18 @@
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'tttesttting6@gmail.com'
+EMAIL_HOST_PASSWORD = 'testing123!'
```
Use environment variables for EMAIL_HOST_USER and EMAIL_HOST_PASSWORD

brazilian-code committed:
Introduced Environment Variables

As pointed out I have switched the sensitive information into the environment variables defined at Heroku

brazilian-code committed:
Added Email Environment Variables

Added email environment variables
